### PR TITLE
Added Class::Load, the missing package before installing the engine

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ This instruction covers the following operating systems:
    > "sudo" (the default is "local::lib", which you do not want).
 
    ```sh
-   sudo cpan -i Locale::TextDomain Hash::Merge Net::IP::XS Zonemaster::LDNS Moose
+   sudo cpan -i Class::Load Locale::TextDomain Hash::Merge Net::IP::XS Zonemaster::LDNS Moose
```
Comment (mattias-p) [inline docs/Installation.md]: We should probably specify the rpm package instead for stability reasons.

**Comment by matsduf** [comment]
@sandoche2k, when you installed Moose with cpan, didn't you get Class::Load at the same time?

**Comment by sandoche2k** [comment]
@matsduf I followed the installation instructions. And i could not continue installation until Class::Load was installed. I will be again running a new installation test and get back to you.

**Comment by matsduf** [comment]
@sandoche2k, please what OS and version? What happend?

**Comment by sandoche2k** [comment]
@matdsuf 
Version : CentOS Linux release 7.3.1611 (Core)

A partial error report
```sh
..... Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/4 subtests 
t/type_constraints/with-specio.t ................................. Can't locate Sub/Exporter.pm in @INC (@INC contains: /root/.cpan/build/Moose-2.2006-byV0D_/blib/lib /root/.cpan/build/Moose-2.2006-byV0D_/blib/arch /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3 /root/perl5/lib/perl5/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3 /root/perl5/lib/perl5/x86_64-linux-thread-multi /root/perl5/lib/perl5 /root/perl5/lib/perl5/5.16.3/x86_64-linux-thread-multi /root/perl5/lib/perl5/5.16.3 /root/perl5/lib/perl5/x86_64-linux-thread-multi /root/perl5/lib/perl5 /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at /root/.cpan/build/Moose-2.2006-byV0D_/blib/lib/Test/Moose.pm line 7.
BEGIN failed--compilation aborted at /root/.cpan/build/Moose-2.2006-byV0D_/blib/lib/Test/Moose.pm line 7.
Compilation failed in require at t/type_constraints/with-specio.t line 5.
BEGIN failed--compilation aborted at t/type_constraints/with-specio.t line 5.
t/type_constraints/with-specio.t ................................. Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
                                                                      t/type_constraints/with-type-tiny.t .............................. skipped: Test requires module 'Types::Standard' but it's not found
t/zzz-check-breaks.t ............................................. 1/3 Warning: Moose did not compile at /root/perl5/lib/perl5/Dist/CheckConflicts.pm line 184.
# Conflicts detected for Module::Runtime:
#   Moose is version unknown, but must be greater than version 2.1202
                                                                      t/zzz-check-breaks.t ............................................. ok

```sh

**Comment by matsduf** [comment]
@sandoche2k, I have tried to reproduce that problem that you report. I followed the instructions word by word on an CentOS  7 server. 

```
$ rpm --query centos-release
centos-release-7-4.1708.el7.centos.x86_64
```
```
$ cat /etc/redhat-release 
CentOS Linux release 7.4.1708 (Core) 
$ cat /etc/system-release
CentOS Linux release 7.4.1708 (Core) 
```

I see no problem with the installation instructions.

```
(...)
t/recipes/basics_genome_overloadingsubtypesandcoercion.t ......... ok    
(...)
Running make test
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'inc', 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t ............ 1/? # Testing Zonemaster Engine v2.0.1, Perl 5.016003, /usr/bin/perl
t/00-load.t ............ ok   
t/asn.t ................ ok   
t/config.t ............. ok    
t/dnsname.t ............ ok    
t/logger.t ............. ok    
t/nameserver-axfr.t .... ok   
t/nameserver.t ......... ok    
t/old-bugs.t ........... ok    
t/pod-coverage.t ....... skipped: Need Pod::Coverage to run these tests.
t/pod.t ................ skipped: Test::Pod 1.22 required for testing POD
t/recursor.t ........... ok    
t/Test-address.t ....... ok    
t/Test-basic.t ......... ok    
t/Test-connectivity.t .. ok    
t/Test-consistency.t ... ok    
t/Test-delegation.t .... ok    
t/Test-dnssec-more.t ... ok    
t/Test-dnssec.t ........ ok     
t/Test-lastcases.t ..... ok   
t/Test-nameserver.t .... ok    
t/Test-syntax.t ........ ok    
t/Test-zone.t .......... ok    
t/translator.t ......... ok   
t/undelegated.t ........ ok    
t/util.t ............... ok   
t/zone.t ............... ok     
t/zonemaster.t ......... ok    
All tests successful.
Files=27, Tests=838, 234 wallclock secs ( 0.16 usr  0.04 sys + 230.64 cusr  2.23 csys = 233.07 CPU)
Result: PASS
  ZNMSTR/Zonemaster-Engine-v2.0.1.tar.gz
  /bin/make test -- OK
Running make install
Installing /usr/local/share/perl5/Zonemaster/Engine.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Util.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/DNSName.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Logger.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Nameserver.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Config.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Overview.pod
Installing /usr/local/share/perl5/Zonemaster/Engine/ASNLookup.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/NSArray.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Recursor.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Constants.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Translator.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Zone.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Packet.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Exception.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/TestMethods.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Address.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Basic.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Delegation.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Connectivity.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Zone.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Nameserver.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/DNSSEC.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Syntax.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Example.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Test/Consistency.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Nameserver/Cache.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Net/IP.pm
Installing /usr/local/share/perl5/Zonemaster/Engine/Logger/Entry.pm
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/iana-profile.json
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/config.json
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/iana-ipv4-special-registry.csv
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/config_example_logfilter.json
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/policy.json
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/iana-ipv6-special-registry.csv
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/locale/en/LC_MESSAGES/Zonemaster-Engine.mo
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/locale/fr/LC_MESSAGES/Zonemaster-Engine.mo
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/locale/fr_FR.UTF-8/LC_MESSAGES/Zonemaster-Engine.mo
Installing /usr/local/share/perl5/auto/share/dist/Zonemaster-Engine/locale/sv/LC_MESSAGES/Zonemaster-Engine.mo
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Address.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Basic.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Util.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Nameserver.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Zone.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Connectivity.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Delegation.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::DNSName.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::DNSSEC.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Logger.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Nameserver.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Config.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Net::IP.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Nameserver::Cache.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::ASNLookup.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Overview.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::NSArray.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Syntax.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Logger::Entry.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Example.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Constants.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Recursor.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Test::Consistency.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Translator.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Packet.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Zone.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::Exception.3pm
Installing /usr/local/share/man/man3/Zonemaster::Engine::TestMethods.3pm
Appending installation info to /usr/lib64/perl5/perllocal.pod
  ZNMSTR/Zonemaster-Engine-v2.0.1.tar.gz
  sudo /bin/make install  -- OK

```

**Comment by sandoche2k** [comment]
Will close it for now as both @matsduf and @mattias-p could not reproduce it.

**Pull request closed without merging**